value-processor/src/org/immutables/value/processor/Immutables.generator

```diff
@@ -2670,6 +2670,9 @@ static final [output.linesShortable]class Json[type.generics]
   [signature] { throw new UnsupportedOperationException(); }
 [/for]
 [for v in type.allAccessibleAttributes]
+  [if v.isGenerateDerived]
```

Comment:
should we mark all stub methods as ignored (not only derived), does it cause any problem with deserialization of mandatory attributes?

Reply:
I don't know for sure. 

I think it only matters, if they are whatever Jackson considers 'getters' in this case.  Since the `class Json` is for deserialization.

https://fasterxml.github.io/jackson-annotations/javadoc/2.5/com/fasterxml/jackson/annotation/JsonIgnore.html

I don't know enough about the lazy attributes, I haven't used them, if they need it or not.